awx/main/management/commands/cleanup_host_metrics.py

```diff
@@ -0,0 +1,22 @@
+from awx.main.models import HostMetric
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    """
+    Run soft-deleting of HostMetrics
+    """
+
+    help = 'Run soft-deleting of HostMetrics'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--months-ago', type=int, dest='months-ago', action='store', help='Threshold in months for soft-deleting')
+
+    def handle(self, *args, **options):
+        months_ago = options.get('months-ago') or None
+
+        if not months_ago:
```

Comment:
This prevents me from doing `--months-ago=0` should that be allowed?
I can also do `-1`, do you want to do any kind of validation on it?

Reply:
it's not supposed to be 0. Negative values will be checked in the `cleanup_task` method